### ISSUE TITLE
Final Report Review by hy488

### ISSUE THREAD

**Issue by haoqiyao** [kathleenzhu/MedicalNoshow#11]
The project is to predict whether a patient will show up on the medical appointment. The team collected data from Kaggle competition and they add weather features to the data. The conclusion they got is that the models they selected generally performed 70% accuracy and they suggested to investigated the neighborhood and collected the data.

The data cleaning part is clear and well-explained. Data manipulation approach is backed up with reasons. There are many plots for showing the relationship between no-show proportions and other features and the team tried to convey how they select their features to the readers. For the model selection, they provided many models with different loss functions and regularizers, and also tree models. At the end, they compared different models' accuracy and gave sufficient explanation.

First suggestion I would give is that the team should perform cross validation to make sure the model does not overfit. Second suggestion is that the model formulas really takes a huge part of the paper. As the viewers are ORIE 4741 students, these formulas can be partially deleted. The final suggestion is that as the model does not fit well since it makes all prediction as "on time", maybe like you said you should investigate more features on it to make the models less trivial.

In conclusion, even though the prediction is not what the team want, they did a great job on it as the procedure is clear and models they use are reasonable.
